src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier(Color.values());
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100) + 1);
```

Comment:
Do not use magic numbers, make a constants!

Reply:
i dont understand

Reply:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants